lib/order.rb

```diff
@@ -0,0 +1,19 @@
+class Order
```

Comment:
You could require the file for the menu class here for ease of use in feature testing.